### PR TITLE
Burnham multi version config

### PR DESCRIPTION
**Pull request by jrbenny35**
This PR adds the configuration necessary to run a Burnham dag against multiple versions of Burnham. It adds an additional client based on the original clients. These new clients will pull the latest Major version of burnham, as specified within the DAG.

### CODE REVIEW

**Review by hackebrot — CHANGES_REQUESTED** [review]
As discussed on Zoom today, a few of the expected query results need to be updated. Clearing this review request for now.

**Comment by jrbenny35** [comment]
These commits are kinda weird but we can fix with a squash

File: dags/burnham.py
Comment (hackebrot) [inline dags/burnham.py:184]: From what I can tell we should expect `1` additional `discovery` ping, see https://github.com/mozilla/telemetry-airflow/pull/1382#pullrequestreview-784554475.

```suggestion
    {"experiment": "spore_drive:tardigrade", "document_count": 3},
```

**Review by hackebrot — CHANGES_REQUESTED** [review]

File: dags/burnham.py
Comment (jrbenny35) [reply dags/burnham.py:184]: Whoops, I had that but maybe it got lost when I had to do a rebase/merge. Just going to add it via github.

**Review by hackebrot — APPROVED** [review]
LGTM! 🚀

**Comment by haroldwoo** [comment]
I would squash the commits so the log doesnt show 17 for this PR. Aftwards, feel free to hit the 'rebase and merge' button

**Comment by jrbenny35** [comment]
I can't merge as I do not have write access.

**Comment by haroldwoo** [comment]
Granted you access

**Pull request merged**
